plugins-compat-tester/src/main/java/org/jenkins/tools/test/model/MavenPom.java

```diff
@@ -64,28 +58,47 @@ private MavenPom(File rootDir, String pomFileName){
 		this.pomFileName = pomFileName;
 	}
 
-	public void transformPom(MavenCoordinates coreCoordinates) throws PomTransformationException{
+	public void transformPom(MavenCoordinates coreCoordinates, boolean testJenkinsVersion) throws PomTransformationException{
 		File pom = new File(rootDir.getAbsolutePath()+"/"+pomFileName);
 		File backupedPom = new File(rootDir.getAbsolutePath()+"/"+pomFileName+".backup");
 		try {
+			Document doc = new SAXReader().read(pom);
 			FileUtils.rename(pom, backupedPom);
-
-			Source xmlSource = new StreamSource(backupedPom);
-            // TODO switch to DOM4J for simplicity and consistency
-			Source xsltSource = new StreamSource(new ClassPathResource("mavenParentReplacer.xsl").getInputStream());
-			Result result = new StreamResult(pom);
-			
-			TransformerFactory factory = TransformerFactory.newInstance();
-			Transformer transformer = factory.newTransformer(xsltSource);
-			transformer.setParameter("parentArtifactId", coreCoordinates.artifactId);
-			transformer.setParameter("parentGroupId", coreCoordinates.groupId);
-			transformer.setParameter("parentVersion", coreCoordinates.version);
-			transformer.transform(xmlSource, result);
+            
+            Element parent = doc.getRootElement().element("parent");
```

Comment:
There's a mix of tabs and spaces in this file... I'll fix the indentation of this whole method, but don't want to modify other parts of the class.